### DISCUSSION TITLE
Praise

### DISCUSSION THREAD

**Discussion by CaptainKirb49** [YoshiCrafter29/YoshiCrafterEngine#44] [General]
Can we all just take a second and praise YoshiCrafter for creating this godsend of an engine. This is the single most well done engine that I have ever had the pleasure of using!

I could rant more but I have mods to make but again, a massive thank you and I can't wait to see how much this engine grows!

Thank you, this is CaptainKirb signing off

PS. Blackjack is going to be on Yoshi Engine

**Comment by NoobManWasTaken** [comment]
yoshi engine best engine

**Reply by CaptainKirb49** [reply]
Based

**Reply by Theinkyslayer** [reply]
True

**Comment by isakube** [comment]
god this engine is wonderful, i was expecting a shitty psych fork but got all this and im at the good loss for words
